src/components/product-search/components/ProductSearchEmbed.vue

```diff
@@ -559,4 +564,23 @@ const preSubmitChecks = (e) => {
         color: $color-base-text;
         opacity: 0.8;
     }
+
+    .product-search__type {
+        @include media-breakpoint-up(lg) {
+            .d-lg-flex {
+                .vs-select {
+                    width: 100%;
+                }
+
+                .product-search__in {
+                    margin-right: $spacer-4;
```

Comment:
Small thing but this could be consolidated into just margin: 0 $spacer-4; ... Just thinking of performance and wherever we can save code we should 😄